src/components/feedback-form.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+// import Form from "./form";
+import styles from "./call-to-action.module.css";
+import PropTypes from "prop-types";
+import Form from "./form";
+
+
+function FeedbackForm() {
+
+  const additionalFormAttr = {
+    "data-netlify": "true",
+    method: "post"
+  }
+  return (
+    <Form
+      className={`flex flex-col rounded shadow w-full mb-4 lg:w-1/2 lg:mr-4 p-6`}
+      additionalFormAttr={additionalFormAttr}
+    >
+      <div>
+        <input type="hidden" netlify-honeypot="bot-field"/>
+        <label>
+          Title:
+          <input className="w-full p6 shadow rounded p-1 mb-3" placeholder="Hell FSN!" type="text" />
```

Comment:
I think you have a typo here. I think you probably wanted to use `Hello FSN!` instead of `Hell FSN!`

```suggestion
          <input className="w-full p6 shadow rounded p-1 mb-3" placeholder="Hello FSN!" type="text" />
```

If you're feeling edgy and want to use `Hell FSN!` I suggest we double down and use `Go to hell FSN!` 😂 jk jk

Reply:
😅😂